web/src/main/java/com/marklogic/hub/web/service/EntityManagerService.java

```diff
@@ -134,10 +134,7 @@ public EntityModel saveEntity(EntityModel entity) throws IOException {
         else {
             HubEntity renamedEntity = em.saveEntity(hubEntity, true);
             entity.setFilename(renamedEntity.getFilename());
-            dataHubService.installUserModules(hubConfig, true, null, null);
```

Comment:
Isn't this still needed for loading the entity model file into staging/final? Granted, it should only load that as opposed to loading all user modules.